pkg/loader/load_test.go

```diff
@@ -384,13 +384,43 @@ func (s *execDDLSuite) TestShouldUseDatabase(c *check.C) {
 	mock.ExpectExec("CREATE TABLE").WillReturnResult(sqlmock.NewResult(0, 0))
 	mock.ExpectCommit()
 
-	loader := &loaderImpl{db: db, ctx: context.Background()}
+	loader := &loaderImpl{db: db, ctx: context.Background(), destDBType: "mysql"}
 
 	ddl := DDL{SQL: "CREATE TABLE", Database: "test_db"}
 	err = loader.execDDL(&ddl)
 	c.Assert(err, check.IsNil)
 }
 
+func (s *execDDLSuite) TestOracleTruncateDDL(c *check.C) {
+	db, mock, err := sqlmock.New()
+	c.Assert(err, check.IsNil)
+
+	mock.ExpectBegin()
+	mock.ExpectExec("BEGIN test.do_truncate").WillReturnResult(sqlmock.NewResult(0, 0))
+	mock.ExpectCommit()
+
+	loader := &loaderImpl{db: db, ctx: context.Background(), destDBType: "oracle"}
+
+	ddl := DDL{SQL: "truncate table t1", Database: "test", Table: "t1"}
+	err = loader.execDDL(&ddl)
+	c.Assert(err, check.IsNil)
+}
+
+func (s *execDDLSuite) TestTruncateTablePartitionDDL(c *check.C) {
+	db, mock, err := sqlmock.New()
+	c.Assert(err, check.IsNil)
+
+	mock.ExpectBegin()
+	mock.ExpectExec("BEGIN test.do_truncate").WillReturnResult(sqlmock.NewResult(0, 0))
+	mock.ExpectCommit()
+
+	loader := &loaderImpl{db: db, ctx: context.Background(), destDBType: "oracle"}
+
+	ddl := DDL{SQL: "alter table t1 truncate partition p1", Database: "test", Table: "t1"}
+	err = loader.execDDL(&ddl)
+	c.Assert(err, check.IsNil)
+}
```

Comment:
What's these two tests 🤔

Reply:
one for "truncate table xx" statement, another is for "alter table xx truncate partition xx"

Reply:
We don't have DDL part now and these two ddls look different. I think we can add these tests later.

Reply:
OK